tests/rkt_caps_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"os"
+	"testing"
+
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/ThomasRooney/gexpect"
+)
+
+var capsTests = []struct {
+	rktCmd string
+	expect string
+}{
+	{
+		`../bin/rkt --debug --insecure-skip-verify run ./rkt-inspect-print-caps.aci`,
+		"Effective: 0xfdecafff",
```

Comment:
It was the number I got on my laptop when I tried it...
On Semaphore, the output is: `Effective: 0xffffffe0fdecafff` but it fills the unexisting caps with 1 instead of 0 for some reasons... Semaphore has a kernel 3.13 with /proc/sys/kernel/cap_last_cap=36 and my laptop has a kernel 3.19 with cap_last_cap=37. Cap37 is CAP_AUDIT_READ.

I don't know how Go can convert this 0xfdecafff number in text. `captest --text` and `pscap -a` can do this easily so I am tempted to remove this test from rkt-inspect.aci and just use the other tests with `pscap -a` below.

Note that the current version of the tests with `pscap -a` just expect what I currently get on my laptop: I have checked whether it is the correct thing.